app/res/values-v11/themes.xml

```diff
@@ -81,6 +81,8 @@
         <item name="android:paddingLeft">@dimen/title_paddingLeft</item>
 	</style>
     
-    
+    <style name="Theme.Dialog" parent="@android:style/Theme.Holo.Light.Dialog">

```

Comment:
This was Richard's change, I don't actually know why it's here. Should I remove?